### PR TITLE
Add collection name and method to emitted data for MongoDB

### PR DESCRIPTION
**Pull request by kenspirit**
#278

### CODE REVIEW

File: probes/mongo-probe.js [hunk 1]
```diff
@@ -101,24 +107,25 @@ MongoProbe.prototype.attach = function(name, target) {
  * Lightweight metrics probe for MongoDB queries
  * 
  * These provide:
- * 		time:		time event started
- * 		query:		the query itself
- * 		duration:	the time for the request to respond
+ *         time:        time event started
+ *         query:        the query itself
+ *         duration:    the time for the request to respond
```
Comment (seabaylea) [inline probes/mongo-probe.js]: Could you also add the inclusion of collectionnName to the comments?

**Comment by seabaylea** [comment]
At first pass of looking at the code, the changes look good. Can you also update the README.md to add the new data into the documentation?

**Comment by kenspirit** [comment]
@seabaylea Sure.  Sorry for missing that.  I have supplemented the documentation in method and README.  BTW, I rename the metrics from collectionName to collection which looks more concise.

**Comment by tobespc** [comment]
@kenspirit before we can accept your pull request, please can you fill out a CLA agreement https://github.com/RuntimeTools/appmetrics/blob/master/CONTRIBUTING.md

**Comment by kenspirit** [comment]
@tobespc Can I fill out and print it as PDF?  Or must I make a hard copy and scan?
[NodeApplicationMetrics-V1-CLA.pdf](https://github.com/RuntimeTools/appmetrics/files/420744/NodeApplicationMetrics-V1-CLA.pdf)

**Comment by tobespc** [comment]
@kenspirit a pdf will be fine

**Comment by kenspirit** [comment]
@tobespc So is the one I attached ok?  I fill it out already.

**Comment by NikCanvin** [comment]
Thanks @kenspirit the CLA looks good, I'll take the fact you commented it with your gitlab id as a signature (all be it a digital one).

**Comment by kenspirit** [comment]
@NikCanvin Thx very much.

**Comment by kenspirit** [comment]
@NikCanvin May I know when this PR can be merged and which version it can include?  I also want to change the appmetrics-elk for this change.

**Comment by NikCanvin** [comment]
@kenspirit , we're very short of people right now, but we'll aim to get the PR merged within days (I assume you can then build appmetrics from github and start using it that way).. as for the next release (cutting a version on NPM), I'm guessing that's likely to be about 1-3 weeks.

**Comment by kenspirit** [comment]
@NikCanvin ic.  Thank you.

**Comment by sjanuary** [comment]
@seabaylea - ok if I merge this? We have a README update now and it would be good to get this PR into 1.0.14.

**Comment by sjanuary** [comment]
As we don't have any automated tests for the probe I have tested this manually and it is working for me, although as a side note I don't think this probe gets all the Mongo information that you would want to see (e.g. we miss quite a few methods including insertMany, insertOne, updateOne)  Probably need to address that separately though.

**Comment by seabaylea** [comment]
I'm a fan of incremental development, so we can merge this as-is and do the further work separately.

**Comment by sjanuary** [comment]
Merged and raised #296 for missing methods.

**Pull request merged**